tests/sentry/api/test_client_state.py

```diff
@@ -0,0 +1,48 @@
+from django.urls import reverse
+
+from sentry.testutils import APITestCase
+
+
+class ClientStateTest(APITestCase):
+    def setUp(self):
+        super().setUp()
+        self.login_as(user=self.user)
+        self.org = self.create_organization(owner=self.user, name="baz")
```

Comment:
Nit: No need to create an org here, you can just use the fixture one already provided

Reply:
fixed, thanks!